tests/test_pytorch/test_accelerator.py

```diff
@@ -399,7 +400,8 @@ def test_hpu_parallel_reduce_op_strategy_default():
     assert strategy.reduce_op == "sum"
 
 
-@pytest.mark.skip(reason="TBD : Fix pytest issues")
+@pytest.mark.standalone()
+@pytest.mark.skip(reason="fix ddp pytest issue")
```

Comment:
This is still skipped...

Reply:
parameterized dist test have issues. We still need a mechanism to run them in separate sessions. As of now enabling some tests in  #119